arlas-core/src/main/java/io/arlas/server/core/FluidSearch.java

```diff
@@ -205,6 +205,10 @@ private BoolQueryBuilder filter(Expression expression, String dateFormat) throws
             throw new InvalidParameterException(INVALID_PARAMETER_F);
         }
         String field = expression.field;
+        FieldMD fieldMD = getFieldStatus(field);
```

Comment:
I don't really understand what is the difference between fieldMD and fieldStatus, why is not always the same?

Reply:
I first called it FieldStatus before renaming it to FieldMD. I forgot to rename it everywhere